src/Engine/ProtoCore/AssociativeGraph.cs

```diff
@@ -16,6 +16,291 @@ public enum UpdateStatus
     public class Utils
     {
         /// <summary>
+        /// Builds the dependencies within the list of graphNodes
+        /// </summary>
+        /// <param name="graphNodeScopeToCheck"></param>
+        public static void BuildGraphNodeDependencies(List<AssociativeGraph.GraphNode> graphNodesInScope)
```

Comment:
Makes sense, Ill note this down as a sub-task to remove the Utils namespace. It is currently used in a lot of places: 
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7433